codecov.yml

```diff
@@ -9,6 +9,7 @@ ignore:
   - "**/styled.tsx"
   - "**/constants/**/*"
   - "constants/**/*"
+  - "src/components/FeatureFlagModal/DevFlagsBox.tsx"
```

Comment:
instead of one-offing a file in here, can you mv it to a dev-specific folder? eg `src/components/dev/...` or `src/dev/DevFlagsBox`?